services/cdnjs/cdnjs.js

```diff
@@ -15,8 +20,10 @@ module.exports = class Cdnjs extends BaseJsonService {
          status code = 200, body = {} */
       throw new NotFound();
     }
-    const version = json.version || 0;
+    return json.version || 0;
+  }
 
+  static formatBadge(version) {
```

Comment:
I really enjoyed working with this abstraction in #1743, where I realized how well it mapped onto React's `render()` function, which at its purest, converts the passed-in properties into a DOM-like structure. To harmonize with that, I'd suggest we pass in an object with a single named property, e.g. `static formatBadge({ version })`.

There's one thing that feels slightly off about this abstraction, which is that you want the render/formatter function to take only the properties it needs, whereas the server response might either be organized in an inconvenient way, or contain a bunch of other stuff. In the npm badges, it worked beautifully to pass a copy of the response object, pruned down to the keys used by the badge. That would work here too. For a service which returned a more poorly organized response object, we'd probably want to clean up the schema before passing it to the render/formatter function.

What do you think about using React's name, `render()`?